hardware/MQTT.cpp

```diff
@@ -482,8 +482,22 @@ void MQTT::on_message(const struct mosquitto_message* message)
 		}
 		else if (szCommand == "sendnotification")
 		{
-			std::string subject, body, sound;
+			uint64_t idx=0;
+			std::string name=std::string(""), subsystems=NOTIFYALL, subject, body, extradata=std::string(""), sound;
```

Comment:
Could you place subsystem on a separate line as you initialize this
You do not have to use std::string(""), a string is initialized empty by default.